runtime/src/main/jni/v8_inspector/src/inspector/v8-network-agent-impl.cpp

```diff
@@ -0,0 +1,74 @@
+//
+// Created by pkanev on 2/22/2017.
+//
+
+#include <assert.h>
+#include "v8-network-agent-impl.h"
+
+namespace v8_inspector {
+
+namespace NetworkAgentState {
+static const char networkEnabled[] = "networkEnabled";
```

Comment:
I've simply used the same convention that was in place by the default v8 inspector implementations that came from the chromium project -> https://github.com/NativeScript/android-runtime/blob/master/runtime/src/main/jni/v8_inspector/src/inspector/v8-profiler-agent-impl.cc#L24

https://chromium.googlesource.com/v8/v8.git/+/refs/heads/5.5.372/src/inspector/v8-runtime-agent-impl.cc#52